plenum/__init__.py

```diff
@@ -35,13 +36,20 @@ def setup_plugins():
         raise ImportError('Incorrect plugin root {}. No such package found'.
                           format(plugin_root))
     enabled_plugins = config.ENABLED_PLUGINS
+    installed_packages = {p.project_name: p for p in pip.get_installed_distributions()}
```

Comment:
Does it work when a plugin is installed from deb package?

Reply:
I second this concern

Reply:
I have not tried it with a deb package but since deb package also places the files in python path, it should work